libs/ui-lib/lib/common/components/operators/operatorSpecs.tsx

```diff
@@ -83,12 +83,27 @@ export const isOCPVersionEqualsOrMore = (
   );
 };
 
+export const highlightMatch = (text: string, searchTerm?: string): React.ReactNode => {
+  if (!searchTerm) return text;
+  const regex = new RegExp(`(${searchTerm})`, 'gi');
```

Comment:
this may crash the app. Ie if the `searchTerm` results in an invalid regex `like `)``. You will also need to handle other special chars like `. *` ..